whylog_vim/output_formater/output_agregator.py

```diff
@@ -0,0 +1,31 @@
+from functools import partial
+from whylog_vim.consts import Messages
+
+
+class OutputAgregator():
+    def __init__(self):
+        self.buttons = {}
+        self.output_lines = []
+
+    def add(self, element):
+        self.output_lines.append(element)
+
+    def add_commented(self, content):
+        self.add(Messages.PREFIX % content)
+
+    def get_content(self):
+        # The same type as vim buffor.
+        return self.output_lines
+
+    def create_button(self, function, **params):
+        line = self._get_line()
```

Comment:
Do I understand it right that we can bind buttons only to the last added line? it is surprising. Wouldn't it be better to have possibility to bind buttons to any chosen line? You can parametrize `create_button` by line_number.